### DISCUSSION TITLE
Open a browser directly in the sandboxie plus

### DISCUSSION THREAD

**Discussion by Kevin5789** [sandboxie-plus/Sandboxie#4137] [General]
Hi can i set a browser for opening directly on sandboxie plus?

**Comment by offhub** [comment]
##### Always (ForceProcess):
> Right click on a sandbox > Sandbox Options > Program Control > Force Programs > Force Program

Select the program or type its name. (e.g. program.exe)

#### Create Shortcut:

1. Expand the sandbox list.

2. Right-click the program for which you want to create a shortcut.

3. Select "Create Shortcut" from the menu that opens.

Similarly, you can open the "`Preset`" submenu and select "`Force into this sandbox`" to always (ForceProcess) open the application in this sandbox.

**Reply by Kevin5789** [reply]
> ##### Always (ForceProcess):
> > Right click on a sandbox > Sandbox Options > Program Control > Force Programs > Force Program
> 
> Select the program or type its name. (e.g. program.exe)
> 
> #### Create Shortcut:
> 1. Expand the sandbox list.
> 2. Right-click the program for which you want to create a shortcut.
> 3. Select "Create Shortcut" from the menu that opens.
> 
> Similarly, you can open the "`Preset`" submenu and select "`Force into this sandbox`" to always (ForceProcess) open the application in this sandbox.

thank you, if i want to have 2 shortcuts of the same browser,  1 that will open inside sandboxie plus e 1 that will open normally without sandboxie how can i do it?

**Reply by offhub** [reply]
#### If you want to start the browser outside the sandbox:

1. Duplicate the shortcut or modify the existing one.
2. Right-click on the shortcut and select `Properties`.
3. Change the command line in the `Target` box.

##### Target:
```
"C:\Program Files\Sandboxie-Plus\Start.exe" /disable_force "C:\Program Files\Browser\browser.exe"
```

4. Click OK to save.

---
#### Example:

Shortcut # 1 (Inside the sandbox):
```
"C:\Program Files\Sandboxie-Plus\Start.exe" /box:BoxName "C:\Program Files\Browser\browser.exe"
```

Shortcut # 2 (Outside the sandbox):
```
"C:\Program Files\Sandboxie-Plus\Start.exe" /disable_force "C:\Program Files\Browser\browser.exe"
```

---

See [Sandboxie Start Command Line](https://sandboxie-plus.github.io/sandboxie-docs/Content/StartCommandLine.html#disable-forced-programs)

**Reply by Kevin5789** [reply]
> #### If you want to start the browser outside the sandbox:
> 1. Duplicate the shortcut or modify the existing one.
> 2. Right-click on the shortcut and select `Properties`.
> 3. Change the command line in the `Target` box.
> 
> ##### Target:
> ```
> "C:\Program Files\Sandboxie-Plus\Start.exe" /disable_force "C:\Program Files\Browser\browser.exe"
> ```
> 
> 4. Click OK to save.
> 
> #### Example:
> Shortcut # 1 (Inside the sandbox):
> 
> ```
> "C:\Program Files\Sandboxie-Plus\Start.exe" /box:BoxName "C:\Program Files\Browser\browser.exe"
> ```
> 
> Shortcut # 2 (Outside the sandbox):
> 
> ```
> "C:\Program Files\Sandboxie-Plus\Start.exe" /disable_force "C:\Program Files\Browser\browser.exe"
> ```
> 
> See [Sandboxie Start Command Line](https://sandboxie-plus.github.io/sandboxie-docs/Content/StartCommandLine.html#disable-forced-programs)

thank you
